apps/gallery/style/gallery.css

```diff
@@ -257,6 +257,11 @@ body.pickView #thumbnails {
   outline-offset: -0.2rem;
 }
 
+/* RTL modifications */
+html[dir=rtl] #thumbnails-number-selected {
+  direction: rtl;
+}
+
```

Comment:
I just stumbled upon this, for consistency with the rest of Gaia css files and the way RTL is done, it's better to have everything-rtl at the bottom of the file. :)